reconnecting_ftp/__init__.py

```diff
@@ -112,7 +112,14 @@ def connect(self) -> None:
         if self.connection is None:
             conn_refused = None  # type: Optional[ConnectionRefusedError]
             try:
-                self.connection = self.FTP(timeout=self.timeout, encoding=self.encoding)
+                try:
```

Comment:
What do you think about directly checking the Python version? This would make the intent more explicit and help the debugging in the future.

```suggestion
                if sys.version_info >= (3, 9):
                    self.connection = self.FTP(timeout=self.timeout, encoding=self.encoding)
                else:
                    self.connection = self.FTP(timeout=self.timeout)
```